test/math.jl

```diff
@@ -1168,6 +1168,12 @@ end
             @test hypot(x, x*f, x*f) ≈ x * hypot(one(f), f, f) rtol=eps(T)
         end
     end
+    @testset "Real and Complex $T" for T in (Float16, Float32, Float64, BigFloat)
+        let x = sqrt(floatmax(T)/2 + 1)
+            @test (@inferred hypot(x, complex(x/4))) ≈ x * sqrt(17/BigFloat(16))
```

Comment:
any reason you used `complex(x/4)` rather than just `x`?

Reply:
its not any special reason, actually it could be just ```x```